apps/src/code-studio/pd/form_components/FieldGroup.story.jsx

```diff
@@ -51,22 +51,22 @@ TestWrapper.propTypes = {
 };
 
 export default {
-  title: 'FormComponents/FieldGroup',
+  title: 'FormComponents/FieldGroup', // eslint-disable-line storybook/no-title-property-in-meta
   component: FieldGroup,
 };
 
 const Template = args => <TestWrapper {...args} />;
 
-export const basicExample = Template.bind({});
-basicExample.args = {
+export const BasicExample = Template.bind({});
```

Comment:
I didn't know this was called pascal case!

Reply:
haha me neither until I saw the error messages :)